README.md

```diff
@@ -108,21 +109,21 @@ surface_texture.present();
 
 See the [`examples/`](https://github.com/linebender/vello/tree/main/examples) folder to see how that code integrates with frameworks like winit and bevy.
 
-
 ## Performance
 
 We've observed 177 fps for the paris-30k test scene on an M1 Max, at a resolution of 1600 pixels square, which is excellent performance and represents something of a best case for the engine.
 
 More formal benchmarks are on their way.
 
-
 ## Integrations
 
 ### SVG
 
-[`vello_svg`](https://github.com/linebender/vello_svg/), supports converting a [`usvg`](https://crates.io/crates/usvg) `Tree` into a Vello scene.
+A separate integration for rendering SVG files is available through the [`vello_svg`](https://github.com/linebender/vello_svg) crate.
+
+This repository also includes the [`demo_svg`](./integrations/demo_svg/) integration, which supports a minimal subset of SVG features needed for vello examples.
```

Comment:
This link doesn't resolve on main or on your branch.

Reply:
Yeah, I missed this when resolving the conflicts. Thanks.